### DISCUSSION TITLE
Vector<T> CrossProduct

### DISCUSSION THREAD

**Discussion by Finnepinnen** [mathnet/mathnet-numerics#839] [Ideas]
Hi

The DotProduct, OuterProduct and ConjugateDotProduct are available but the CrossProduct isnt and I'm starting to doubt my reading skills. Can it be that the CrossProduct is missing? 

**2012**
https://stackoverflow.com/questions/11759720/cross-product-using-math-net-numerics-with-c-sharp

**2016**
https://discuss.mathdotnet.com/t/how-to-use-cross-product/125

**Comment by cdrnet** [comment]
Yes, the cross product is intentionally missing, since the types in the LinearAlgebra namespace of Math.NET Numerics are inteded for linear algebra, not (spatial) geometry. In linear algebra, the concept of a cross product doesn't make much sense, and it is defined for length-3 verctors only. Meanwhile, many aspects of linear algebra are not useful in geometry.

For (spatial) geometry we recommend to use Math.NET Spatial instead, where the cross product is available, along with other geometry functionaility.
